.github/workflows/release.yml

```diff
@@ -1,39 +1,23 @@
-name: sonar-release
-# This workflow is triggered when publishing a new github release
+name: release
```

Comment:
Existing release was moved to draft. That might be OK.
Existing tag was deleted from the release - is that expected?

Reply:
There was no correlation, I recreated the release to retrigger the action and did a poor job at that, sorry for the noise